test/wrappers/test_lambda.js

```diff
@@ -7,8 +7,30 @@ const consts = require('../../src/consts.js');
 const awsLambdaTrigger = require('../../src/triggers/aws_lambda.js');
 const lambdaWrapper = require('../../src/wrappers/lambda.js');
 const errorCode = require('../../src/proto/error_code_pb.js');
+const config = require('../../src/config.js');
+
+const RETURN_VALUE = { result: 1 };
+
+// Helpers functions
+function getRunner(addEventStub) {
+    const calls = addEventStub.getCalls();
+    for (let i = 0; i < calls.length; i += 1) {
+        const event = calls[i].args[0];
+        if (event.getOrigin && event.getOrigin() === 'runner') {
+            return event;
+        }
+    }
+    return null;
+}
+
+function getReturnValue(addEventStub) {
+    const runnerMetadata = getRunner(addEventStub).getResource().getMetadataMap();
+    return runnerMetadata.get('return_value');
+}
 
 describe('lambdaWrapper tests', () => {
+    before(() => { config.setConfig({ metadataOnly: false }); });
```

Comment:
please put it in the before each (i know currently its not needed but if someone adds a test that changes the config we will be fucked)
on another thought - please mock getConfig - so the config we get is local to the test and won't be affected by other tests

Reply:
This whole `config` usage is very bad. What you suggest is just a small patch so I think I should just leave it as it.